src/PointwiseFunctions/GeneralRelativity/ProjectionOperators.hpp

```diff
@@ -44,8 +44,8 @@ void transverse_projection_operator(
  * \ingroup GeneralRelativityGroup
  * \brief Compute projection operator onto an interface
  *
- * \details Returns the operator \f$P_{ij} = \gamma_{ij} - n_i n_j \f$,
- * where \f$ \gamma_{ij} \f$ is the spatial metric, and \f$ n_i\f$ is
+ * \details Returns the operator \f$P_{ij} = g{ij} - n_i n_j \f$,
+ * where \f$ g{ij} \f$ is the spatial metric, and \f$ n_i\f$ is
```

Comment:
you lost the `_` to make the indices downstairs on this line and the line above

Reply:
argh, fixed!